components/Avatar/style.css

```diff
@@ -0,0 +1,50 @@
+.avatar{
+    border-radius:50%;
+  }
+  .square-avatar{
+    border-radius:0.5rem;
+  }
+  .avatar-sm{
+    width:3rem;
+  }
+  .avatar-md{
+    width:5rem;
+  }
+  .avatar-lg{
+    width:8rem;
+  }
+  .letter-avatars{
+  
```

Comment:
You could avoid this empty line and rather leave one empty line after each class. That helps in code readability.

Reply:
okay i resolve that